tests/unittests/parsing/PreprocessorTests.cpp

```diff
@@ -1326,6 +1326,30 @@ TEST_CASE("unconnected_drive directive") {
     CHECK(!diagnostics.empty());
 }
 
+TokenKind lexCellDefine(std::string_view text) {
```

Comment:
Looks like this function return type should be `bool`

Reply:
Fixed